MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ComboBox.xaml

```diff
@@ -26,6 +26,7 @@
   <converters:FloatingHintOffsetCalculationConverter x:Key="FloatingHintOffsetCalculationConverter" />
   <converters:ComboBoxClearButtonMarginConverter x:Key="ComboBoxClearButtonMarginConverter" />
   <converters:DoubleToThicknessConverter x:Key="DoubleToThicknessConverter" />
+  <converters:DoubleToThicknessConverter x:Key="DoubleToThicknessConverterWithNegativeOffset" InitialOffset="-8" />
```

Comment:
Can we add a comment on this converter saying where this -8 value comes from.

Reply:
Sure thing

Reply:
Added comment